test/functional/shared/cli/cli_test.go

```diff
@@ -66,31 +66,37 @@ func verifyRecipeCLI(ctx context.Context, t *testing.T, test shared.RPTest) {
 	// for AWS test and has the AWS scope which the environment created in this does not.
 	envName := test.Steps[0].RPResources.Resources[0].Name
 	recipeName := "recipeName"
-	recipeTemplate := "ghcr.io/testpublicrecipe/bicep/modules/testTemplate:v1"
-	templateKind := "bicep"
-	resourceType := "Applications.Datastores/mongoDatabases"
-	file := "testdata/corerp-redis-recipe.bicep"
+	recipe1 := "recipe1"
+	recipe2 := "recipe2"
+	registry := strings.TrimPrefix(functional.GetBicepRecipeRegistry(), "registry=")
+	version := strings.TrimPrefix(functional.GetBicepRecipeVersion(), "version=")
```

Comment:
Why do we need to do this?

Reply:
These functions prefix the registry and version strings so we need to remove them for this test. https://github.com/radius-project/radius/blob/c262df7aee22cf1ebea2fa66d3e70cfdaa135ca0/test/functional/testUtil.go#L91